<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="CL-NB-944.local" ignored="0" name="CursoAppium" tests="1" failures="1" timestamp="04 ene 2021 04:30:26 GMT" time="45.582" errors="0">
  <testcase name="test" time="34.523" classname="testSuites.EjecucionCasos">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 5, Size: 5">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 5, Size: 5
at java.util.ArrayList.rangeCheck(ArrayList.java:659)
at java.util.ArrayList.get(ArrayList.java:435)
at io.appium.java_client.pagefactory.ElementListInterceptor.getObject(ElementListInterceptor.java:40)
at io.appium.java_client.pagefactory.interceptors.InterceptorOfAListOfElements.intercept(InterceptorOfAListOfElements.java:49)
at $java.util.ArrayList$$EnhancerByCGLIB$$b5855c01.get(<generated>)
at pages.CarruselPage.validarCliente(CarruselPage.java:146)
at testClases.cliente.TestCaseUltimaClase.frontEnd(TestCarrusel.java:24)
at testSuites.EjecucionCasos.test(examplesDriver.java:27)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- CursoAppium -->
